tests/include/test/drivers/size.h

```diff
@@ -29,50 +29,54 @@
 #if defined(PSA_CRYPTO_DRIVER_TEST)
 #include <psa/crypto_driver_common.h>
 
-/** \def TEST_KEY_CONTEXT_BASE_SIZE
+/** \def TEST_DRIVER_KEY_CONTEXT_BASE_SIZE
  *
  * This macro returns the base size for the key context. It should at a minimum
- * be at least 20 bytes, allowing for integers for each field.
+ * 20 bytes.
  */
-#define TEST_KEY_CONTEXT_BASE_SIZE          20
+#define TEST_DRIVER_KEY_CONTEXT_BASE_SIZE          20
 
-/** \def TEST_KEY_CONTEXT_KEY_PAIR_SIZE
+/** \def TEST_DRIVER_KEY_CONTEXT_KEY_PAIR_SIZE
  *
  * Number of bytes included in every key context for a key pair.
  */
 
-#define TEST_KEY_CONTEXT_KEY_PAIR_SIZE      0
+#define TEST_DRIVER_KEY_CONTEXT_KEY_PAIR_SIZE      0
 
-/** \def TEST_KEY_CONTEXT_PUBLIC_KEY_SIZE
+/** \def TEST_DRIVER_KEY_CONTEXT_PUBLIC_KEY_SIZE
  *
  * Number of bytes included in every key context for a public key.
  */
-#define TEST_KEY_CONTEXT_PUBLIC_KEY_SIZE    0
+#define TEST_DRIVER_KEY_CONTEXT_PUBLIC_KEY_SIZE    0
 
-/** \def TEST_KEY_CONTEXT_SYMMETRIC_FACTOR
+/** \def TEST_DRIVER_KEY_CONTEXT_SYMMETRIC_FACTOR
  *
  * Every key context for a symmetric key includes this many times the key size.
  */
-#define TEST_KEY_CONTEXT_SYMMETRIC_FACTOR   0
+#define TEST_DRIVER_KEY_CONTEXT_SYMMETRIC_FACTOR   0
 
-/** \def TEST_KEY_CONTEXT_STORE_PUBLIC_KEY
+/** \def TEST_DRIVER_KEY_CONTEXT_STORE_PUBLIC_KEY
  *
  * If this is true for a key pair, the key context includes space for the public key.
  * If this is false, no additional space is added for the public key.
  */
-#define TEST_KEY_CONTEXT_STORE_PUBLIC_KEY   0
+#define TEST_DRIVER_KEY_CONTEXT_STORE_PUBLIC_KEY   0
 
-/** \def TEST_KEY_CONTEXT_SIZE_FUNCTION
+/** \def TEST_DRIVER_KEY_CONTEXT_SIZE_FUNCTION
  *
- * If this is defined a size_function is implemented.
+ * The test driver supports a size_function() implementation
```

Comment:
What about:
"If TEST_DRIVER_KEY_CONTEXT_SIZE_FUNCTION is defined, the test driver provides a size_function entry point, otherwise, it does not." ?